fbw-a380x/src/systems/instruments/src/SD/Pages/ElecDc/elements/TransformerRectifier.tsx

```diff
@@ -84,7 +84,11 @@ export const TransformerRectifier: FC<TransformerRectifierProps> = ({ x, y, bus
         {Math.round(potential)}
       </text>
       {/* Current */}
-      <text x={65} y={97} className={`F28 EndAlign ${trCurrentNormal ? 'Green' : 'Amber'} LS1`}>
+      <text
+        x={65}
+        y={97}
+        className={`F28 EndAlign ${trCurrentNormal ? 'Green' : title === 'APU TR' ? 'White' : 'Amber'} LS1`}
```

Comment:
I strongly doubt that this should actually be white. The image provided in the reference issue is not really conclusive when looking at, and using a color picker tool, there is clear green in the area surrounding the letter